docs/en/query_language/system.md

```diff
@@ -85,4 +86,20 @@ Enables background data distribution when inserting data into distributed tables
 SYSTEM START DISTRIBUTED SENDS [db.]<distributed_table_name>
 ```
 
+
+### STOP|START MERGES {#query_language-system-stop-start-merges}
+
+Provides possibility to stop and start background merges for all tables in the MergeTree family:
+
+```sql
+SYSTEM STOP|START MERGES
+```
+
+Optionally we can define specific table:  
+
+```sql
+SYSTEM STOP|START MERGES [db.]<merge_tree_family_table_name>
+```
+NOTE: DETACH / ATTACH table will start background merges for the table even in case we stopped merges for all *MergeTree tables before).
```

Comment:
```suggestion
!!! note "Note":
    `DETACH / ATTACH` table will start background merges for the table even in case when merges have been stopped for all *MergeTree tables before.
```